### PR TITLE
Update main_tool_and_resource_list.csv

### PR DESCRIPTION
**Pull request by Drosophilic**
2nd attempt. Now with the csv updated as requested.

### CODE REVIEW

**Comment by bedroesb** [comment]
@Drosophilic I am wondering wwhy the workflow complains, I will get back on you about this

File: _data/main_tool_and_resource_list.csv [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
+IntAct,http://www.ebi.ac.uk/intact/,"IntAct provides a freely available open source database system and analysis tools for protein interaction data.",fairsharing:d05nwx,"share, re-use"
+Metabolights,http://www.ebi.ac.uk/metabolights,"MetaboLights is a database for metabolomics studies their raw experimental data and associated metadata.",fairsharing:kkdpxe,"share, re-use"
+PRIDE,http://www.ebi.ac.uk/pride,"The PRIDE PRoteomics IDEntifications database is a centralized standards compliant public data repository that provides protein and peptide identifications together with supporting evidence.",fairsharing:e1byny,"share, re-use"
+PDBE,http://www.ebi.ac.uk/pdbe/,"The Protein Data Bank in Europe (PDBe) is the European resource for the collection organisation and dissemination of data on biological macromolecular structures.",fairsharing:26ek1v,"share, re-use"
+EGA,https://ega-archive.org/,"The European Genome-phenome Archive (EGA) allows you to deposit and explore datasets from molecular studies on human subjects",fairsharing:mya1ff,"share, re-using, human data, privacy"
```
Comment (bedroesb) [inline _data/main_tool_and_resource_list.csv:33]:
```suggestion
EGA,https://ega-archive.org/,"The European Genome-phenome Archive (EGA) allows you to deposit and explore datasets from molecular studies on human subjects",fairsharing:mya1ff,"share, re-use, human data, privacy"
```

**Comment by bedroesb** [comment]
Is their a reason to take out the commas?

**Comment by Drosophilic** [comment]
I'm not sure which commas you are referring to. 

The workflow seemed to complain as excel had messed up some of the formatting.

**Comment by bedroesb** [comment]
the switch from xlsx to csv is quite recent. If you open the files changed tab you see that most of the commas in the description disappear. I can maybe push an update where the descriptions are between double quotes for better excel compliance?

**Comment by Drosophilic** [comment]
That would certainly help. I think I chose the hardest and less efficient way of doing the update, so next time should be easier :)

**Comment by bedroesb** [comment]
this is a very good test in general, so it takes up more of your time than probably anticipated!

**Comment by bedroesb** [comment]
The thing that still surprises me is that also within the double quotes, the commas get deleted...

**Comment by Drosophilic** [comment]
It's all learning for next time.

**Comment by bedroesb** [comment]
I just see that opening it on linux with libre office doesn't give a problem, while excel on windows does.

**Comment by bedroesb** [comment]
@Drosophilic I saw there where some characters that were misinterpreted by excel on windows. I think I fixed those in my latest commit. Could please retry to apply you changes onto the new table? SOorry for all the hassle! The csv table was build up fro a google doc spreadsheet an apparently some weird characters appeared.

**Comment by Drosophilic** [comment]
FYI, I was using a mac.

**Comment by bedroesb** [comment]
Ok good to know, I was wondering if you open the new table on mac, make changes on it and save it again, whether you lose again the commas in the descriptions... It is also possible they get lost while importing them into excel.

**Comment by Drosophilic** [comment]
OK. I've made some more changes by importing the csv into excel on my mac and then exporting back out.

**Comment by bedroesb** [comment]
did you pull in the latest one? I fixed some characters

**Comment by Drosophilic** [comment]
Sorry, I won't get to this today. I'll try to look later. In the meantime, could my changes be accepted into master?

**Pull request merged**